build-scripts/bundle.js

```diff
@@ -53,13 +54,13 @@ module.exports.babelOptions = ({ latestBuild }) => ({
   babelrc: false,
   presets: [
     !latestBuild && [
-      require("@babel/preset-env").default,
+      "@babel/preset-env",
```

Comment:
we used to do this because of some things didn't manage to resolve. Glad it works now.